README.rst

```diff
@@ -201,6 +201,14 @@ template ``context_processors`` in ``settings.py`` as shown below.
 Settings
 ^^^^^^^^
 
+``TOPOLOGY_API_URI``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**example**: ``http://topology.example.com/api/receive/``
```

Comment:
Why aren't we defining the base url only?
If we do this we will have to define the URL for every similar use cases.
Suppose we add another API that we call in the admin, we'll have to add a new setting for each case.

Can you change the logic so that only the base url is passed and the rest of the URL path is still calculated by the code?

The setting could be named ``TOPOLOGY_API_URL``, I would specify that it defaults to `None` and it's automatically calculated.

Reply:
I see you point. Yes, adding a variable for every URL sounds bad. 
However, if the user doesn't mention `/api/receive/`, I'll have to hard code it, that sucks too!

Reply:
-- Made the changes --